code/datums/black_market_sellable.dm

```diff
@@ -69,78 +71,361 @@ var/list/black_market_sellables = list()
+	desc = "Looking for NT-brand explosive valves. The tanks attached are irrelevant, send it empty if you want."
+	demand_min = 1
+	demand_max = 2
+	price_min = 400
+	price_max = 500
+	sps_chance = 70
+	display_chance = 50
+
+/datum/black_market_sellable/weapons/ied
+	name = "Improvised Explosive Device"
+	item = /obj/item/weapon/grenade/iedcasing/
+	no_children = 1
+	desc = "Looking for cheap explosives made from soda cans or something."
+	demand_min = 3
+	demand_max = 7
+	price_min = 50
+	price_max = 120
+	sps_chance = 5
+	display_chance = 80
 	
+/datum/black_market_sellable/weapons/dermal
+	name = "Dermal Armour Patch"
+	item = /obj/item/clothing/head/helmet/tactical/HoS/dermal
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 300
+	price_max = 350
+	sps_chance = 30
+	display_chance = 40	
 	
 	
 	
+/datum/black_market_sellable/machinery
+	category = "Machinery and Technology"
 	
+/datum/black_market_sellable/machinery/janicart
+	name = "Nanotrasen-Brand Janicart"
+	item = /obj/structure/bed/chair/vehicle/janicart
+	no_children = 1
+	desc = "Don't need the key or anything."
+	demand_min = 1
+	demand_max = 1
+	price_min = 300
+	price_max = 350
+	sps_chance = 35
+	display_chance = 35
+
+/datum/black_market_sellable/machinery/plasmaminer
+	name = "Plasma Gas Miner"
+	item = /obj/machinery/atmospherics/miner/toxins
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 250
+	price_max = 300
+	sps_chance = 20
+	display_chance = 30
 	
+/datum/black_market_sellable/machinery/sleepingminer
+	name = "N2O Gas Miner"
+	item = /obj/machinery/atmospherics/miner/sleeping_agent
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 125
+	price_max = 175
+	sps_chance = 10
+	display_chance = 30
 	
+/datum/black_market_sellable/machinery/planningframe
+	name = "AI Law Module Planning Frame"
+	item = /obj/item/weapon/planning_frame
+	no_children = 1
+	demand_min = 1
+	demand_max = 1
+	price_min = 200
+	price_max = 300
+	sps_chance = 40
+	display_chance = 45	
+
 	
 	
+
+/datum/black_market_sellable/animals
+	category = "Living Creatures"	
 	
+/datum/black_market_sellable/animals/slime
+	name = "? Slime"
+	item = /mob/living/carbon/slime/adult
+	no_children = 0
+	desc = "Slime must be an adult."
+	demand_min = 1
+	demand_max = 3
+	price_min = 200
+	price_max = 400
+	sps_chance = 0
+	display_chance = 60
+	teleport_modifier = 1.5
+	var/list/potential_slimes = list("sepia","adamantine","pyrite","bluespace","cerulean")
+	var/selected_slime
 	
+/datum/black_market_sellable/animals/slime/purchase_check(var/obj/input, var/mob/user)
+	if(istype(input,/mob/living/carbon/slime/adult))
+		var/mob/living/carbon/slime/adult/input_slime = input
+		if(input_slime.colour == selected_slime)
+			return VALID
+	return "Our buyer is looking for a [selected_slime] slime, not whatever the hell that is."
 	
+/datum/black_market_sellable/animals/slime/on_setup()
+	selected_slime = pick(potential_slimes)
+	name = capitalize(selected_slime) + " Slime"
```

Comment:
Don't have to capitalize it if you capitalize the slime pick names in potential_slimes

Reply:
actually just try to test and see if capitalized potential_slimes works just as well for detecting slime colors

Reply:
It doesn't, the slime colors are non-capitalized and capitalized == comparisons won't work in slime code.